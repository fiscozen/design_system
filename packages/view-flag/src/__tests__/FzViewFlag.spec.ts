import { describe, it } from "vitest";
import { mount } from "@vue/test-utils";
import { FzViewFlag } from "..";

describe.concurrent("FzViewFlag", () => {
  it("matches snapshot", async ({ expect }) => {
    const wrapper = mount(FzViewFlag, {
      props: {
        environment: "staging",
        firstName: "Mario",
        lastName: "Rossi",
        role: "Operatore",
      },
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
