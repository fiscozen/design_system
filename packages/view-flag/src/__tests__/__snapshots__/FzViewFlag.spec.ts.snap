// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`FzViewFlag > matches snapshot 1`] = `
"<div class="border-8 border-semantic-warning fixed inset-0 grid">
  <div class="h-32 bg-semantic-warning justify-self-center self-end rounded-t-base text-sm flex p-6 gap-12 items-center">
    <div class="text-xs px-12 rounded-xl w-fit h-20 flex items-center font-medium bg-core-black text-core-white">staging</div>
    <div>Operatore: Mario R.</div>
  </div>
</div>"
`;
